<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Identification</class>
 <widget class="QMainWindow" name="Identification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Identification</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>991</width>
      <height>691</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>310</y>
       <width>291</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="iLEditNomUtilisateur">
        <property name="placeholderText">
         <string>Nom d'utilisateur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="iLEditMotDePasse">
        <property name="placeholderText">
         <string>Mot de passe</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>430</y>
       <width>291</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="iBtnSeConnecter">
        <property name="text">
         <string>Se connecter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="iBtnAnnuler">
        <property name="text">
         <string>Annuler</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="iLabEtat">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>530</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat de la connexion</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
